src/commands/squad_markup.py

```diff
@@ -0,0 +1,46 @@
+import disnake
+from disnake.ext import commands
+
+
+class Ps2SquadMarkup(commands.Cog):
+    """
+    Class cog for ps2 squad markup help message.
+    """
+
+    @commands.slash_command(dm_permission=True)
+    async def squad(self, inter):
+        pass
+
+    @squad.sub_command()
+    async def markup(
+            self,
+            inter: disnake.ApplicationCommandInteraction
+    ):
+        """
+        Print the markup for colourful squad names.
+        """
+        await inter.response.defer(ephemeral=True)
+
+        markup = {
+            "Alpha": "`<font color=\"#ff0000\">✵ FU Alpha Platoon/Voice Lead ✴</font>`",
+            "Bravo": "`<font color=\"#ff0000\">✵ FU Bravo Platoon/Voice Lead ✴</font>`",
+            "Charlie": "`<font color=\"#ff0000\">✵ FU Charlie Platoon/Voice Lead ✴</font>`",
+            "Delta": "`<font color=\"#ff0000\">✵ FU Delta Platoon/Voice Lead ✴</font>`"
```

Comment:
I'd suggest making stars the same on both sides of text
```suggestion
            "Alpha": "`<font color=\"#ff0000\">✵ FU Alpha Platoon/Voice Lead ✵</font>`",
            "Bravo": "`<font color=\"#ff0000\">✵ FU Bravo Platoon/Voice Lead ✵</font>`",
            "Charlie": "`<font color=\"#ff0000\">✵ FU Charlie Platoon/Voice Lead ✵</font>`",
            "Delta": "`<font color=\"#ff0000\">✵ FU Delta Platoon/Voice Lead ✵</font>`"
```